osctiny/extensions/packages.py

```diff
@@ -343,7 +343,7 @@ def cmd(self, project, package, cmd, **params):
 
         return response.text
 
-    def checkout(self, project, package, destdir, rev=None, meta=False):
+    def checkout(self, project, package, destdir, rev=None, meta=False, expand=0):
```

Comment:
Just a little change to indicate that this should be considered as a boolean flag
```suggestion
    def checkout(self, project, package, destdir, rev=None, meta=False, expand=False):
```